src/aig-common/old-common/services/cpv.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { Resolve } from '@angular/router';
+import { Injectable } from '@angular/core';
+import { HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { CpvService } from './cpv.service';
+
+type EntityArrayResponseType = HttpResponse<any[]>;
+
+@Injectable()
+export class CpvResolver implements Resolve<Observable<EntityArrayResponseType>> {
+  constructor(
+    private cpvService: CpvService
```

Comment:
not use this service but CpvResourceService, take ispiration from this file: [https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/generic/resolver/eopoo-type.resolver.ts](url)